schema/project-level/project-schema.json

```diff
@@ -1426,6 +1435,32 @@
       "required": [
         "id"
       ]
+    },
+    "Person": {
+      "title": "Person",
+      "description": "A natural person.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "Identifier",
+          "description": "A local identifier for this person within the people array.",
```

Comment:
Will the Person object always be used in a people array? If not, then this should be made more generic.

Reply:
Good point, probably best to allow for reuse. Fixed in https://github.com/open-contracting/infrastructure/pull/279/commits/3397485730d766c0b694496c447cb2cff6de926f